modules/mixed-arch-modify-machine-set.adoc

```diff
@@ -0,0 +1,76 @@
+//Module included in the following assembly
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-modify-machine-set_{context}"]
+
+= Adding a machine set to your cluster using the `arm64` bootimage  
```

Comment:
@aleskandro I combined the procedure in slack and the comments in the PR. How does this updated procedure look?

Reply:
Hi @kelbrown20 added a few other comments, but in general, we're good now.